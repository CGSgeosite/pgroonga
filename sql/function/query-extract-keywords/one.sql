SELECT pgroonga.query_extract_keywords('Groonga');
