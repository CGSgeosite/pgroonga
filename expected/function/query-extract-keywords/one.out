SELECT pgroonga.query_extract_keywords('Groonga');
 query_extract_keywords 
------------------------
 {Groonga}
(1 row)

